### ISSUE TITLE
result file not generated after successfully running the tool on AWS

### ISSUE THREAD

**Issue by shaoyiZ** [AlexandrovLab/SigProfilerExtractor#78]
Hi! Thanks for the video, they were really helpful. 
The extractor tool was run successfully but the program did not generate a file after running. I tried both on a Mac and Windows and I am sure I SSHed into AWS. 

Here are the codes: 
$sudo amazon-linux-extras enable python3.8
%sudo yum install python3.8
$sudo yum install python38-devel gcc
$pip3.8 install --user SigProfilerExtractor

$Python3.8
>>>from SigProfilerMatrixGenerator import install as geninstall
>>>geninstall.install("GRCh37") 

$wget ftp://alexandrovlab-ftp.ucsd.edu/pub/tools/SigProfilerExtractor/Example_data/21BRCA.zip
$unzip 21BRCA
$from SigProfilerExtractor import sigpro as sig
$sig.sigProfilerExtractor("matrix","results","21BRCA/21BRCA.vcf",reference_ge     nome="GRCh37",minimum_signatures=1,maximum_signatures=10,nmf_replicates=100,cpu=-1)

I checked the file where the pem file was located but no luck. Can you please provide some insights to this issue?

I really appreciate it!

**Comment by mishugeb** [comment]
Hi @shaoyiZ ,
Could you please try the following command and see if that is working?
$sig.sigProfilerExtractor("matrix","results","21BRCA/21BRCA.txt",reference_genome="GRCh37",minimum_signatures=1,maximum_signatures=10,nmf_replicates=100,cpu=-1)

Thanks,
Mishu

**Comment by shaoyiZ** [comment]
Dear Mishu, 

Thanks a lot for your reply! I was just able to re-run the codes. I used the code you provided in the last Thread but still no luck. 

I can see that the tool was run successfully since I got the following feedback from the instance. 

Decompositon Plot made for SBS96A
Decompositon Plot made for SBS96B
Decompositon Plot made for SBS96C
Your Job Is Successfully Completed! Thank You For Using SigProfiler Extractor.

But I still cannot find a folder on my laptop that is named results. The pem file was located in a file named AWS on my desktop and I was checking in that folder. Am I looking in the right place? Your video mentioned the file should be in the directory where the instance was started so I was assuming this is the place to look. 

Thanks a lot!
Shaoyi
